packages/avatar/tests/avatar.test.tsx

```diff
@@ -1,7 +1,41 @@
 import * as React from "react"
-import { render, testA11y } from "@chakra-ui/test-utils"
+import { render, testA11y, waitFor } from "@chakra-ui/test-utils"
 import { Avatar, AvatarBadge } from "../src"
 
+const DELAY = 0
+const LOAD_IMAGE = "load"
+const ERROR_IMAGE = "error"
+const orignalImage = window.Image
+
+const mockImage = (loadState: string) => {
```

Comment:
There are only two allowed strings for this function, so we can use a more narrow type for `loadState`.

```suggestion
const mockImage = (loadState: "load" | "error") => {
```